src/dashboard.js

```diff
@@ -780,68 +780,189 @@ function renderHistogramGraph(hgram) {
+function renderHistogramEvolution(lines, minDate, maxDate) {
+  var drawnLines = lines.filter(function(line) { return !line.disabled; });
+  
+  // Filter out the points that are outside of the time range
+  var filteredDatasets = drawnLines.map(function (line) {
+    return {
+      label: line.key,
+      strokeColor: line.color,
+      data: line.values.filter(function(point) { return point.x >= minDate && point.x <= maxDate; }),
+    };
+  })
+  
+  // Plot the data using Chartjs
+  if (gCurrentHistogramEvolutionPlots !== null) {
+    gCurrentHistogramEvolutionPlots.destroy();
+  }
+  var ctx = document.getElementById("evolution").getContext("2d");
+  Chart.defaults.global.responsive = true;
+  gCurrentHistogramEvolutionPlots = new Chart(ctx).Scatter(filteredDatasets, {
+    animation: false,
+    scaleType: "date",
+    scaleLabel: function(valuesObject) { return fmt(valuesObject.value); },
+    tooltipFontSize: 10,
+    tooltipTemplate: function(valuesObject) {
+      return valuesObject.datasetLabel + " - " + fmt(valuesObject.y) + " on " + valuesObject.argLabel;
+    },
+    multiTooltipTemplate: function(valuesObject) {
+      return valuesObject.datasetLabel + " - " + fmt(valuesObject.y) + " on " + valuesObject.argLabel;
+    },
+    bezierCurveTension: 0.3,
+    pointDotStrokeWidth: 0,
+    pointDotRadius: 3,
   });
 }
 
-var renderHistogramTime = null;
-var lastHistogramEvos = null;
-var _exportHgram = null;
-var _lastBlobUrl = null;
+var gPreviousBlobUrl = null;
 // Generate download on mousedown
 $('#export-link').mousedown(function () {
   if (_lastBlobUrl) {
-    URL.revokeObjectURL(_lastBlobUrl);
-    _lastBlobUrl = null;
+    URL.revokeObjectURL(gPreviousBlobUrl);
+    gPreviousBlobUrl = null;
   }
   var csv = "start,\tend,\tcount\n";
-  csv += _exportHgram.map(function (count, start, end, index) {
+  csv += gCurrentHistogram.map(function (count, start, end, index) {
     return [start, end, count].join(",\t");
   }).join("\n");
 
-  _lastBlobUrl = URL.createObjectURL(new Blob([csv]));
-  $('#export-link')[0].href = _lastBlobUrl;
-  $('#export-link')[0].download = _exportHgram.measure() + ".csv";
-  event('click','download csv', 'download csv');
+  gPreviousBlobUrl = URL.createObjectURL(new Blob([csv]));
+  $('#export-link')[0].href = gPreviousBlobUrl;
+  $('#export-link')[0].download = gCurrentHistogram.measure() + ".csv";
+  event('click', 'download csv', 'download csv');
 });
 
-var hasReportedDateRangeSelectorUsedInThisSession = false;
+var gHasReportedDateRangeSelectorUsedInThisSession = false;
+var gDrawTimer = null;
+function updateRendering(hgramEvo, lines) {
+  // Filter dates
+  var dates = hgramEvo.dates();
+  var start = dates[0], end = dates[dates.length - 1];
+
+  // Update the start and end range and update the selection if necessary
+  var picker = $("#dateRange").data("daterangepicker");
+  var startMoment = moment(start), endMoment = moment(end);
+  picker.setOptions({
+    format: "MM/DD/YYYY h:mm A",
```

Comment:
I don't think we need time for the picker.. maybe just a tooltip to explain that the range is inclusive
In particular, the Nightly with buildID "May 8" is generated shortly after midnight of May 9th based on Eastern time.. so let's not mention times